magefile.go

```diff
@@ -114,6 +113,25 @@ var (
 	goIntegTestTimeout = 2 * time.Hour
 	// goProvisionAndTestTimeout is the timeout used for both provisioning and running tests.
 	goProvisionAndTestTimeout = goIntegTestTimeout + 30*time.Minute
+
+	helmChartsValues = []struct {
+		path        string
+		versionKeys []string
+		tagKeys     []string
+	}{
+		// elastic-agent Helm Chart
+		{
+			"./deploy/helm/elastic-agent",
```

Comment:
nit: let's stick with the `const helmChartPath`

Reply:
Makes sense, fixed in https://github.com/elastic/elastic-agent/pull/5822/commits/16fdafc196f65b36864840084db15499b0e99ed3